Updates and final review of How To

EvaKaul opened the pull request:
- few updates of content
- final review comments

vasilogi commented:
Generally, I could follow all the steps clearly except the part concerning the **Configuring application** section that speaks about the two ways of adding a configuration file. The rest is clear.

EvaKaul commented:
Hi @vasilogi, 
I updated the how to according to your comments!

Comment 1: Done

Comment 2: I added an additional sentence for clarification "Here the notification source name was set to "KPI calculation app", which can be later used within other API calls."

Comment 3: I changed the whole configuration chapter and added more information

Comment 4: Done. For your issue "...when I had faulty = 7 and produced = 17, i.e. KPI = 100 - (7/17)*100 = 59 the app did not push any notification..." => therefore I added an additional hint at the end. I think that was the problem in your case.

Comment 1 (FlowCreator): I left the deprecated request since some Edge users already asked about it, e.g. im Yammer

Comment 1 (Postman): I added a description and an image how to get the authToken out of the cookies.

If everything is fine for you, I will merge the pull request. Just give me the OK from your side!

vasilogi commented:
Great job 👍 You can merge
